### PR TITLE
[plugins/aws][feat] Add support for SQS

### PR DESCRIPTION
**Pull request by anjafr**
# Description

# To-Dos

- [x] Add test coverage for new or updated functionality
- [x] Lint and test with `tox`
- [ ] Document new or updated functionality (someengineering/resoto.com#XXXX)

- Fixes #XXXX

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://resoto.com/code-of-conduct).

### CODE REVIEW

File: plugins/aws/resoto_plugin_aws/resource/sqs.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    @classmethod
+    def collect(cls: Type[AwsResource], json: List[Json], builder: GraphBuilder) -> None:
+        def add_tags(queue: AwsSqsQueue) -> None:
+            tags = builder.client.list("sqs", "list-queue-tags", result_name="Tags", QueueUrl=[queue.sqs_queue_url])
+            if tags:
+                queue.tags = cast(Dict[str, Optional[str]], tags)
+
+        for queue_url in json:
+            queue_attributes = builder.client.call(
```
Comment (meln1k) [inline plugins/aws/resoto_plugin_aws/resource/sqs.py:65]: Wdyt about not doing this call here, but rather using an approach similar to add_tags? this way we won't block the current thread for the http call.

Comment (anjafr) [reply plugins/aws/resoto_plugin_aws/resource/sqs.py:65]: I don't get an approach like that to work . With only the `QueueUrl` from the first `list-queues` api call it's not possible to create an instance with `instance = cls.from_api(js)` 🤔

Comment (meln1k) [reply plugins/aws/resoto_plugin_aws/resource/sqs.py:65]: I was thinking about something like:
```
for queue_url in json:
    builder.submit_work(add_instance, queue_url)
    ...
 ```
 
 and move the content of the existing loop into the `add_instance` method

File: plugins/aws/tools/model_gen.py [hunk 2]
```diff
@@ -616,6 +616,11 @@ def all_models() -> List[AwsModel]:
     "service-quotas": [
         # AwsResotoModel("list-service-quotas", "Quotas", "ServiceQuota", prefix="Quota", prop_prefix="quota_")
     ],
+    "sqs": [
+        AwsResotoModel(
```
Comment (meln1k) [inline plugins/aws/tools/model_gen.py:620]: please comment it out :)

**Pull request merged**